src/ecs/transform/bundle.rs

```diff
@@ -0,0 +1,52 @@
+//! ECS transform bundle
+
+use app::ApplicationBuilder;
+use ecs::ECSBundle;
+use ecs::transform::*;
+use error::Result;
+
+/// Transform bundle
+///
+/// Will register transform components, and the TransformSystem.
+/// TransformSystem will be registered with name "transform_system".
+///
+/// # Errors
+///
+/// No errors will be returned by this bundle.
+///
+/// # Panics
+///
+/// Panics in TransformSystem registration if the bundle is applied twice in the same dispatcher.
+///
+pub struct TransformBundle<'a> {
+    dep: &'a [&'a str],
+}
+
+impl<'a> TransformBundle<'a> {
+    /// Create a new transform bundle
+    pub fn new() -> Self {
+        Self { dep: &[] }
+    }
+
+    /// Set dependencies for the TransformSystem
+    pub fn with_dep(mut self, dep: &'a [&'a str]) -> Self {
```

Comment:
This is something that makes the new bundle system more complex and gives the user less control. Also see my comment above. But well, let's see how it works out.

Reply:
You are not forced to use the bundle system, you can do all it does on it's own, and the code is open. 
I feel this is a way of lowering the threshold to start using the engine, without giving anything up really.